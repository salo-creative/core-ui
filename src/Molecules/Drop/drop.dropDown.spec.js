import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../test';

// COMPONENT
import DropDown from './drop.dropDown';

test('Renders against snapshot', async () => {
  const { container } = renderWithTheme(
    <DropDown element={ {
      clientHeight: 100,
      clientWidth: 100,
      getBoundingClientRect: () => {
        return {
          top: 100,
          left: 100,
          bottom: 100
        };
      }
    } }
    />
  );
  expect(container.firstChild).toMatchSnapshot();
});