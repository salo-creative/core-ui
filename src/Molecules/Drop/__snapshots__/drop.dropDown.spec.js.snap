// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders against snapshot 1`] = `null`;
